### PR TITLE
chore: backport en.json

### PR DESCRIPTION
**Pull request by mroz22**
This is a minimalistic version of "backport en.json" task

It does:
- take strings from en.json and put them into messages.ts
- add a script that generates new messages (rawMessages.ts) from messages.ts and en.json

Next steps: 
- sync new source strings to crowding.

### CODE REVIEW

File: packages/suite/src/support/rawMessages.ts [hunk 1]
```diff
@@ -0,0 +1,6496 @@
...
+    TR_404_TITLE: {
+        defaultMessage: 'Error 404: Link not found',
+        id: 'TR_404_TITLE',
+    },
+    TR_ACCESS_HIDDEN_WALLET: {
+        defaultMessage: 'Access Hidden wallet',
+        id: 'TR_ACCESS_HIDDEN_WALLET',
+    },
+    TR_WALLET_SELECTION_ACCESS_HIDDEN_WALLET: {
+        defaultMessage: 'Access Hidden wallet\n',
```
Comment (matejkriz) [inline packages/suite/src/support/rawMessages.ts]: There is `\n` at the end of many messages. I believe, that it is just unintentional extra enter in Crowdin message. What about to filter them out in the script?

Comment (mroz22) [reply packages/suite/src/support/rawMessages.ts]: But are we sure that this is not intentional? It might have been intention of the translator.. maybe?

Comment (matejkriz) [reply packages/suite/src/support/rawMessages.ts]: But what would be the intention? To change the layout and add some extra space after? It doesn't work like that, in most cases there is no visible change. 

I believe, that `\n` inside the text is ok, but not at the end.

Comment (mroz22) [reply packages/suite/src/support/rawMessages.ts]: checkout here 214e416b2a4b805b317354363ad6271249e69220

**Review by matejkriz — APPROVED** [review]

**Pull request merged**